ffw/UIRPC.js

```diff
@@ -199,8 +199,29 @@ FFW.UI = FFW.RPCObserver.create(
           case 'UI.SubtleAlert':
           {
             if (SDL.SDLModel.onUISubtleAlert(request.params, request.id)) {
-              SDL.SDLController.onSystemContextChange(request.params.appID);
+              if(!request.params.softButtons){
```

Comment:
@VadymLuchko Please convert this expression to boolean, if its array `!!request.params.softButtons.length`